.github/dependabot.yml

```diff
@@ -1,16 +1,15 @@
 ---
+# Copyright (c) Ansible Project
```

Comment:
Does this config need a GPL license, especially in an MIT project (well, kinda: https://github.com/ansible-community/sphinx_ansible_theme/issues/26)?